sso-kit-starter/src/main/java/com/vaadin/sso/starter/SingleSignOnConfiguration.java

```diff
@@ -51,6 +52,10 @@
 @EnableConfigurationProperties(SingleSignOnProperties.class)
 public class SingleSignOnConfiguration extends VaadinWebSecurity {
 
+    static {
+        UsageStatistics.markAsUsed("sso/SingleSignOnConfiguration", null);
```

Comment:
Is this the desired name for usage statistics? In Collaboration Kit we have `vaadin-collaboration-engine` so I'd expect something like `vaadin-sso-kit` here, like this:

```suggestion
        UsageStatistics.markAsUsed("vaadin-sso-kit", null);
```

Also, it it okay to not specify the version?

Reply:
What if we add the usage statistic to the `LicenseCheckerServiceInitListener`? There we have the product name and version already available, e.g.

```java
public class LicenseCheckerServiceInitListener
        implements VaadinServiceInitListener {

    static final String PROPERTIES_RESOURCE = "sso-kit.properties";

    static final String VERSION_PROPERTY = "sso-kit.version";

    static final String PRODUCT_NAME = "vaadin-sso-kit";

    @Override
    public void serviceInit(ServiceInitEvent event) {
        final var service = event.getSource();

        try {
            final var properties = loadAllProperties(PROPERTIES_RESOURCE);
            final var version = properties.getProperty(VERSION_PROPERTY);

            UsageStatistics.markAsUsed(PRODUCT_NAME, version);

            // Check the license at runtime if in development mode
            if (!service.getDeploymentConfiguration().isProductionMode()) {
                // Using a null BuildType to allow trial licensing builds
                // The variable is defined to avoid method signature ambiguity
                BuildType buildType = null;
                LicenseChecker.checkLicense(PRODUCT_NAME, version, buildType);
            }
        } catch (IOException e) {
            throw new ExceptionInInitializerError(e);
        }
    }
}
```